File: content/polympiads/1.introduction-aux-tableaux/codes/empty.cpp
#include <iostream>
#include <vector>

using namespace std;

int main () {
    int nbElements;
    cin >> nbElements;

    vector<int> tableau(nbElements);
    for (int iElement = 0; iElement < nbElements; iElement ++) {
        cin >> tableau[iElement];
    }
    
    int indiceDepart, indiceFin;
    cin >> indiceDepart >> indiceFin;

    // Implémentez ce que demande la consigne

    for (int iElement = 0; iElement < nbElements; iElement ++) {
        cout << tableau[iElement] << " ";
    }
}

File: content/4.polympiads1/1.introduction-aux-tableaux/.codes/empty.java
import java.util.Scanner;

class Main { 
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
		int nbElements = sc.nextInt();
		int[] tableau  = new int[nbElements];
		for (int i = 0; i < nbElements; i ++) {
			tableau[i] = sc.nextInt();
		}
		
		int indiceDepart = sc.nextInt();
		int indiceFin    = sc.nextInt();
		
		// Implémentez ce que demande la consigne
		
		for (int i = 0; i < nbElements; i ++) {
			System.out.print(tableau[i]);
			System.out.print(" ");
		}
		System.out.println();
	}
}